external-report-demo/report-demo/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "portal-external-report-demo",
+  "version": "1.0.0",
+  "description": "SAM demo of external report",
+  "main": "app.js",
+  "license": "MIT",
+  "dependencies": {
+    "axios": "^0.21.1",
```

Comment:
Is this used anywhere? I didn't see a `require` for it.

Reply:
:+1: